.azure/default-build.yml

```diff
@@ -27,7 +27,7 @@ jobs:
   
   - task: CMake@1
     inputs:
-      cmakeArgs: .. ${{ parameters.cMakeRunArgs }} -DCMAKE_TOOLCHAIN_FILE=../submodules/vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=${{ parameters.configuration }}
+      cmakeArgs: .. ${{ parameters.cMakeRunArgs }} -DCMAKE_TOOLCHAIN_FILE=../submodules/vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=${{ parameters.configuration }} -DUSE_CPPRESTSDK=true
```

Comment:
How is this supposed to be used? If I set USE_CPPRESTSDK to false, how do I provide my own implementation?

Reply:
Doesn't work yet. But ideally if it's false you'll build the library and we'll add runtime checks that you provided http/websockets when making a connection.